cache/1/payloads/0xdAbad81aF85554E9ae636395611C58F7eC1aAEc5.json

```diff
@@ -19,7 +19,10 @@
         "signature": "execute()",
         "callData": "0x"
       }
-    ]
+    ],
+    "id": 0,
```

Comment:
i find these additions weird tbh. as it's just duplications
this is literally the index of the object

Reply:
I need this parameter for comfortable use in the interface

you have to look at the big picture and not just for your needs

Reply:
hmm but i don't understand.

If I am able to access `urls/cache/<chainId>/<address>[proposalId]` I should be perfectly aware about the values of `chainId`, `address` and `proposalId` - because if not, i would not be able to access it in first place.